Bottom: 9d11b5838e5d0033d314a6cbf25d2433abd8af9d
Top:    7f6c835ab7fe0db8d626deb60ac2b318d1547174
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2011-09-03 19:45:33 +0100

Refresh of use-trackball-to-allow-rotatio

---

diff --git a/src/hid/gtk/gui-top-window.c b/src/hid/gtk/gui-top-window.c
index 8168eb5..0df71dd 100644
--- a/src/hid/gtk/gui-top-window.c
+++ b/src/hid/gtk/gui-top-window.c
@@ -1354,7 +1354,8 @@ ghid_build_pcb_top_window (void)
                     G_CALLBACK (ghid_port_rotate), NULL);
   g_signal_connect (trackball, "view-2d-changed",
                     G_CALLBACK (ghid_view_2d), NULL);
-  gtk_box_pack_start (GTK_BOX (vbox_left), trackball, FALSE, FALSE, 0);
+  gtk_box_pack_start (GTK_BOX(ghidgui->left_toolbar),
+                      trackball, FALSE, FALSE, 0);
 #endif
 
   ghidgui->mode_buttons0_frame_vbox = gtk_vbox_new (FALSE, 0);
